.azure/scripts/push-to-central.sh

```diff
@@ -18,6 +18,6 @@ export GPG_TTY=$(tty)
 echo $GPG_SIGNING_KEY | base64 -d > signing.gpg
 gpg --batch --import signing.gpg
 
-GPG_EXECUTABLE=gpg mvn $MVN_ARGS -DskipTests -s ./.azure/scripts/settings.xml -pl ./,api -P ossrh deploy
+GPG_EXECUTABLE=gpg mvn $MVN_ARGS -DskipTests -s ./.azure/scripts/settings.xml -pl ./,api -P central-publishing deploy
```

Comment:
```suggestion
GPG_EXECUTABLE=gpg mvn $MVN_ARGS -DskipTests -s ./.azure/scripts/settings.xml -pl ./,api -P central deploy
```

Reply:
Yeah sorry I didn't see that ... yeah why not :) I guess we can rename the `pushtonexus` Makefile goal in operators repository as well then. 

That being said, +1 for renaming the script (and other occurrences where the `nexus` is).

Reply:
> @im-konge agree ... and what about renaming the push-to-nexus.sh to push-to-central.sh everywhere as Kate did here?

Yeah, sure, why not. +1.

Reply:
Yep happy to be consistent, I've updated the profile to be just `central`